test/Microsoft.ML.Tests/Transformers/TextFeaturizerTests.cs

```diff
@@ -714,6 +714,7 @@ public void LdaWorkout()
         }
 
         [Fact]
+        [Trait("Category", "SkipInCI")]
         public void LdaWorkoutEstimatorCore()
```

Comment:
>LdaWorkoutEstimatorCore [](start = 20, length = 23)

I haven't see this fail in full build and CI build in last 14 days, this should be low fail rate tests and I think it is okay to left this one enabled, if we see this fails often later we can go back to this one